tests/src/Functional/ApigeeEdgeTestTrait.php

```diff
@@ -72,6 +72,7 @@ protected function restoreKey() {
       'active_key_oauth_token' => '',
     ];
     $this->container->get('state')->set('apigee_edge.auth', $keys);
+    drupal_flush_all_caches();
```

Comment:
Clear states cache only with `$this->container->get('state')->resetCache()`

Reply:
For some reason that doesn't do the trick in Drupal 8.6. I spent half a day trying to figure out why.

Reply:
It would be really great of we could figure this out, because it is critical for us.